apps/ecommerce/lambda/src/middlewares/getAppInstallationParameters.ts

```diff
@@ -43,33 +46,34 @@ export const getAppInstallationParametersMiddleware: RequestHandler = async (
           });
         });
 
-      // get installation parameters from via CMA
-      await axios
-        .get(
-          `https://api.contentful.com/spaces/${spaceId}/environments/${environmentId}/app_installations/${appId}`,
+      try {
+        const cmaClient = createClient(
           {
-            headers: {
-              Authorization: `Bearer ${appAccessToken}`,
-              'Content-Type': 'application/json',
+            accessToken: appAccessToken,
+            host,
+          },
+          {
+            type: 'plain',
+            defaults: {
+              spaceId,
+              environmentId,
             },
           }
-        )
-        .then((response) => {
-          const appInstallation = response.data;
-          Object.assign(installationParameters, appInstallation.parameters);
-        })
-        .catch((e) => {
-          console.error(e.message);
-          throw new UnableToGetAppInstallationParameters(
-            'Unable to get app installation parameters',
-            {
-              cause: e,
-            }
-          );
+        );
+
+        const appInstallation = await cmaClient.appInstallation.get({
+          appDefinitionId: appId,
         });
-    }
+        Object.assign(installationParameters, appInstallation.parameters);
 
-    req.installationParameters = installationParameters;
+        req.installationParameters = installationParameters;
```

Comment:
This line must be run outside of the if-condition that starts in line 21. Otherwise we don't set `req.installationParameters` when `parametersHeader` is provided